source/style-guide/style/collapsible-sections.txt

```diff
@@ -0,0 +1,57 @@
+.. _collapsible-sections:
+
+====================
+Collapsible Sections
+====================
+
+.. default-domain:: mongodb
+
+Collapsible sections let you hide and show dynamic content. 
+
+Collapsible Section Anatomy
+---------------------------
+
+A collapsible section contains the following parts:
+
+1. Section title
+2. Short description of the content in the section
+3. Chevron to expand and collapse the section
+4. Section content
+
+.. figure:: /images/collapsible-section-anatomy.png
+   :alt: collapsible section anatomy
+   :align: center
+   :figwidth: 700px
+
+Collapsible Section Syntax
+--------------------------
+
+The following example shows the syntax for a collapsible section:
+
+.. code-block::
+
+   .. collapsible::
+      :heading: Prerequisites
+      :sub_heading: Complete the previous steps in the tutorial and install dependencies
+
+      Body content goes here.
+
+Collapsible Section Guidelines
+------------------------------
+
+When you create collapsible sections, use the following 
+guidelines:
+
+- Make the title short and concise.
+
+- For the subheader, write a very short one-line descrition with a 
+  period at the end. Don't exceed one line.
+
+  .. figure:: /images/collapsible-section-subheader.png
```

Comment:
Note for this image, there is a typo in each example: Prerequisistes -> Prerequisites